client/my-sites/posts/main.jsx

```diff
@@ -56,56 +60,48 @@ const PostsMain = React.createClass( {
 			return false;
 		}
 
+		if ( ! site.jetpack && this.props.author && this.props.myDraftCount === 0 ) {
+			return false;
+		}
+
 		return true;
 	},
 
 	mostRecentDrafts() {
 		const site = this.props.sites.getSelectedSite();
-		const isLoading = this.props.draftCount !== 0 && this.props.loadingDrafts;
 
 		if ( ! site || ! this.showDrafts() ) {
 			return null;
 		}
 
+		const { draftCount, translate } = this.props;
+		const isLoading = draftCount !== 0 && this.props.loadingDrafts;
+
 		return (
 			<div className="posts__recent-drafts">
 				<QueryPosts
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ translate( 'Latest Drafts' ) }>
+					<Button compact href={ this.props.newPostPath }>
+						{ translate( 'Start New' ) }
```

Comment:
I wonder if we should just keep the plus sign here?

@mtias Did you move away from that to make it look better with the column of ellipses following it?